modules/conversion-center/assets/scss/widgets/contact-buttons.scss

```diff
@@ -1,13 +1,17 @@
 .e-contact-buttons {
-    --e-contact-buttons-chat-box-width: 360px;
+    --e-contact-buttons-chat-box-width: #{$contact-buttons-default-width};
     --e-contact-buttons-size-small: 65px;
     --e-contact-buttons-size-medium: 75px;
     --e-contact-buttons-size-large: 85px;
-    --e-contact-buttons-dot: #FF0000;
-    --e-contact-buttons-dot-size: 20px;
-    --e-contact-buttons-profile-dot-bg: #39AA59;
-    --e-contact-buttons-box-shadow: 0 3px 3px 0 rgba(0,0,0,0.25);
-    --e-contact-buttons-border-radius: 7px;
+    --e-contact-buttons-dot: #{$contact-buttons-dots-color};
+    --e-contact-buttons-dot-size: #{$contact-buttons-dots-size};
+    --e-contact-buttons-profile-dot-bg: #{$contact-buttons-profile-dot-bg};
+    --e-contact-buttons-box-shadow: #{$contact-buttons-box-shadow};
+    --e-contact-buttons-border-radius: #{$contact-buttons-border-radius-rounded};
+    --e-contact-button-chat-button-animation-duration: 800ms;
+    --e-contact-button-chat-button-animation-delay: 0;
+    --e-contact-buttons-chat-box-bg: #FFFFFF;
```

Comment:
Suggestion:
Why some of the values are assigned in [contact-buttons-vars.scss](https://github.com/elementor/elementor/pull/26840/files#diff-2cf6296375bbe123bc52901b991a39284a8fbd789ef3d08f77a68996256450ea) and some do not?
Maybe we can unify this?

Reply:
I've cleaned this up. Thanks!